### PR TITLE
refactor(metastore/informer): wrap GetXXX methods error with errdefs

### PR DESCRIPTION
**Pull request by lianxmfor**

### CODE REVIEW

File: internal/database/metadata/informer/informer.go [hunk 1]
```diff
@@ -157,7 +158,7 @@ func (f *Informer) GetRevision(ctx context.Context, id int) (*types.Revision, er
 	if revision := f.Cache().Revisions.Find(func(r *types.Revision) bool {
 		return r.ID == id
 	}); revision == nil {
-		return nil, fmt.Errorf("revision not found")
+		return nil, errdefs.NotFound(fmt.Errorf("revision not found"))
```
Comment (jinghancc) [inline internal/database/metadata/informer/informer.go:161]: add revisionID in error message?

**Review by jinghancc — APPROVED** [review]
LGTM

File: internal/database/metadata/informer/informer.go (hunk 1 above)
Comment (YikSanChan) [inline internal/database/metadata/informer/informer.go:161]: revision %d not found

File: internal/database/metadata/informer/informer.go [hunk 2]
```diff
@@ -167,7 +168,7 @@ func (f *Informer) GetRevisionBy(ctx context.Context, groupID int, revision int6
 	if revision := f.Cache().Revisions.Find(func(r *types.Revision) bool {
 		return r.GroupID == groupID && r.Revision == revision
 	}); revision == nil {
-		return nil, fmt.Errorf("revision not found")
+		return nil, errdefs.NotFound(fmt.Errorf("revision not found"))
```
Comment (YikSanChan) [inline internal/database/metadata/informer/informer.go:171]: revision %d not found

**Review by YikSanChan — APPROVED** [review]

**Review by wfxr — APPROVED** [review]

File: internal/database/metadata/informer/informer.go (hunk 2 above)
Comment (lianxmfor) [reply internal/database/metadata/informer/informer.go:171]: maybe `revision not found by group %d and revision %d` better?

File: internal/database/metadata/informer/informer.go (hunk 1 above)
Comment (lianxmfor) [reply internal/database/metadata/informer/informer.go]: fixed

Comment (lianxmfor) [reply internal/database/metadata/informer/informer.go]: done

**Pull request merged**